### PR TITLE
Update requirements

### PR DESCRIPTION
**Pull request by minhn9753**
Update requirements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
-- Pet's name, customer's email address, and customer postcode are optional in an appointment, however, customers are encouraged to give such information and register to be a user, and als register their pet(s).
-- At the vets, there are also different medicines in stock for treatments at the vet and for customer to buy and perform treatments at home.
+- Pet's name, customer's email address, and customer postcode are optional in an appointment, however, customers are encouraged to give such information and register to be a user, and also register their pet(s).
+- At the vet's, there are also different medicines in stock for treatments at the vet and for customer to buy and perform treatments at home.
 - During the appointments, different health services like X-ray, blood work, special eye check, vaccinations, surgeries,... may be performed. All services have different code and price.
 - When a pet is registered, the pet's council registration number is optional. This will allow future changes of info and integration with the local council system for pets.
+- 1 veterinarian can be booked by many customers.
```
Comment (Claire-LHV) [inline README.md:11]: We can trace this by looking at Appointment entity. There is no need for another direct relation between the two.
We can remove this.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
+- 1 service can be taken by many veterinarians
```
Comment (Claire-LHV) [inline README.md:12]: Change to 1 main responsible vet for a performed service.

[extend] how to track the surgery team, not just the responsible one. eg. Surgery entity <- need something better

Comment (Claire-LHV) [reply README.md:12]: [Minh's Idea] Specialist, General entities

File: README.md [hunk 3]
```diff
@@ -16,9 +19,17 @@
 - A user can pay online for a booking
 - A user can choose the vet for their pet --> constraint: vet needs to be available at the time of the booking. Slot will only be reserved for 5 minutes since the time when the picking is selected.
 - A user can change the pet's information where relevant, and request to change other information where vet staff need to double-check the information before the change.
+- Booking confirmation can only be sent to customers after payment process is successful.
```
Comment (Claire-LHV) [inline README.md:22]: Book Appointment -> vets perform services in the appointment -> bill -> receipt
Customer can pay at the counter right after the appointment, or pay online only for user (registered customer).
Guests must pay at the counter.

File: README.md [hunk 4]
```diff
@@ -16,9 +19,17 @@
...
+- Customers need to register their pets before proceeding to book an appointment.
```
Comment (Claire-LHV) [inline README.md:23]: Not needed.
To encourage pets being registered, receptionists will help with registering their pets anytime.

File: README.md [hunk 5]
```diff
@@ -16,9 +19,17 @@
...
+- Pets need to be linked with customer accounts, so customer accounts need to be registered first in order to register for pet details.
```
Comment (Claire-LHV) [inline README.md:24]: 👍

File: README.md [hunk 6]
```diff
@@ -16,9 +19,17 @@
...
+- Customers can only choose the time slot that is available. If it is fully booked, then customers cannot book.
 
 ## Non-functional requirements identified
 - Not too slow to load
+- There is no booking time slot clash.
```
Comment (Claire-LHV) [inline README.md:29]: not non-functional, codes need to make sure of this -> need validations

File: README.md [hunk 7]
```diff
@@ -16,9 +19,17 @@
...
+- Cannot register a new account with existing email address.
```
Comment (Claire-LHV) [inline README.md:30]: duplicate

File: README.md [hunk 8]
```diff
@@ -16,9 +19,17 @@
...
+- Cannot register new pet with same details.
```
Comment (Claire-LHV) [inline README.md:31]: depends ID instead, and microchip number

File: README.md [hunk 9]
```diff
@@ -16,9 +19,17 @@
...
+- Cannot make duplicate bookings: same time slot and same veterinarian.
```
Comment (Claire-LHV) [inline README.md:32]: Code should check, should not allow second booking to be confirmed. This is functional.

**Review by Claire-LHV — CHANGES_REQUESTED** [review]
Request some changes

File: README.md (hunk 8 above)
Comment (minhn9753) [reply README.md:31]: need to clarify again about ID, microchip number?

**Pull request closed without merging**